editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -209,13 +228,18 @@ void SpriteFramesEditor::_sheet_preview_input(const Ref<InputEvent> &p_event) {
 			if (frames_selected.has(idx)) {
 				frames_selected.erase(idx);
 			} else {
-				frames_selected.insert(idx);
+				frames_selected.insert(idx, selected_count++);
```

Comment:
I'd split this line so that the logic of using and incrementing the property is clear.